src/JWT/JWT.csproj

```diff
@@ -10,20 +10,6 @@
     <TargetFrameworkIdentifier>.NETStandard</TargetFrameworkIdentifier>
   </PropertyGroup>
 
-  <PropertyGroup>
```

Comment:
These and any other project under the /src folder will get these properties from the Directory.Build.props file.

Reply:
But we have just 1 project under /src, should we move anything to a common fille? Won't it hurts readability/maintainability?

Reply:
I did it mostly to increase maintainability since in my experience these settings get out of sync fast when a new project gets introduced

Reply:
I totally agree for general case, and that's what I do at my work projects too. But this project haven't seen more than packageable project since its inception and very unlikely will see it in the future. So I'd like to keep it simple and park for now. As soon if we'll decide to have more than one project - we'll have a common version file.